.envs/.local/.django

```diff
@@ -6,4 +6,6 @@ IPYTHONDIR=/app/.ipython
 
 SECRET_KEY=~R.e|Olx,rMS:m#w%z@ZY!(}C~h>!i~s_sM]v;DrJS+|Lu#]H{
 DJANGO_DEBUG=True
-ALLOWED_HOSTS=['*']
\ No newline at end of file
+ALLOWED_HOSTS=['*']
+
+OPENCAGE_API_KEY=030fb8a03ec845bd858f01e28f092aff
```

Comment:
please drop the key from here & rotate this key as it has been exposed now. Instead could you add a note to the README where we can make a the key for local testing? Link to docs or something?

Reply:
I've made the readme instructions clearer in this pull request:
https://github.com/PolyglotDevsLondon/sharespots/pull/111

The reason I did this is that the project would not build for me without adding the geolocation api key and the instructions weren't clear where to add it.

@abuDarda97 Please can you review this pull request and merge it into development branch?